lib/matplotlib/patches.py

```diff
@@ -3407,8 +3407,7 @@ def transmute(self, path, mutation_size, linewidth):
             return _path, _fillable
 
     class Curve(_Curve):
-        """
-        A simple curve without any arrow head.
+        """A simple curve without any arrow head.
```

Comment:
Yes they don't look good at all. But I think somewhere I saw that for single line docstring.